src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -66,8 +66,10 @@
          (i18n/label :t/colour)]
         [quo/color-picker
          {:default-selected @account-color
-          :on-change        #(reset! account-color %)}]]
-       [rn/view {:style (style/divider-line theme)}]
+          :on-change        #(reset! account-color %)
+          :container-style {:padding-horizontal 12
```

Comment:
Kindly check [this](https://github.com/status-im/status-mobile/pull/17465) PR Omar, We don't want horizontal margins for color picker anymore as it adds bad UX

Reply:
@ibrkhalil this is padding not a margin. It takes full device width